pennylane_lightning/lightning_qubit.py

```diff
@@ -18,17 +18,16 @@
 from typing import List
 from warnings import warn
 from os import getenv
-from itertools import islice
+from itertools import islice, product
+from string import ascii_letters as ABC
```

Comment:
Since `ABC` in Python is normally meant for abstract base classes as `from abc import ABC` we should probably rename this to something else